// @esri/calcite-components/dist/esm/bg-a92d835e.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.12.2
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-b28eded7.js';
import './index-f8bb3936.js';
import './form-b15611bf.js';
import './dom-e42c716f.js';
import './guid-3e8856d2.js';
import './resources-ff1bca76.js';
import './interactive-8550e5bc.js';
import './browser-acc8cea6.js';
import './key-fb0c71ea.js';
import './label-6166a873.js';
import './component-df464e97.js';
import './loadable-cc08b3ee.js';
import './locale-de3ead11.js';
import './observers-81fbf268.js';
import './focusTrapComponent-8e33376b.js';
import './config-86b2d7cb.js';
import './math-e5bddb69.js';
import './t9n-d78203a0.js';
import './Validation-3f102285.js';
import './input-d8caf5d2.js';

// Bulgarian [bg]
var locale = {
  name: 'bg',
  weekdays: 'неделя_понеделник_вторник_сряда_четвъртък_петък_събота'.split('_'),
  weekdaysShort: 'нед_пон_вто_сря_чет_пет_съб'.split('_'),
  weekdaysMin: 'нд_пн_вт_ср_чт_пт_сб'.split('_'),
  months: 'януари_февруари_март_април_май_юни_юли_август_септември_октомври_ноември_декември'.split('_'),
  monthsShort: 'яну_фев_мар_апр_май_юни_юли_авг_сеп_окт_ное_дек'.split('_'),
  weekStart: 1,
  ordinal: function ordinal(n) {
    var last2Digits = n % 100;

    if (last2Digits > 10 && last2Digits < 20) {
      return n + "-\u0442\u0438";
    }

    var lastDigit = n % 10;

    if (lastDigit === 1) {
      return n + "-\u0432\u0438";
    } else if (lastDigit === 2) {
      return n + "-\u0440\u0438";
    } else if (lastDigit === 7 || lastDigit === 8) {
      return n + "-\u043C\u0438";
    }

    return n + "-\u0442\u0438";
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'D.MM.YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY H:mm',
    LLLL: 'dddd, D MMMM YYYY H:mm'
  },
  relativeTime: {
    future: 'след %s',
    past: 'преди %s',
    s: 'няколко секунди',
    m: 'минута',
    mm: '%d минути',
    h: 'час',
    hh: '%d часа',
    d: 'ден',
    dd: '%d дена',
    M: 'месец',
    MM: '%d месеца',
    y: 'година',
    yy: '%d години'
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
